### PR TITLE
Time setting on Raspberry after screenly loading

### PR DESCRIPTION
**Pull request by antonmolodykh**
for #546

### CODE REVIEW

File: viewer.py [hunk 1]
```diff
@@ -315,6 +317,12 @@ def setup():
 
     signal(SIGUSR1, sigusr1)
     signal(SIGUSR2, sigusr2)
+    
+    c = ntplib.NTPClient() 
+    responce = c.request('europe.pool.ntp.org', version=3) 
+    cur_date = ctime(responce.tx_time) 
+    sudodate = subprocess.Popen(["sudo", "date", "-s", cur_date])
```
Comment (vpetersson) [inline viewer.py]: This won't work unfortunately because we cannot assume we can run `sudo` without a password.

**Comment by vpetersson** [comment]
@antonmolodykh I'm not too keen on this approach. Screenly shouldn't communicate with the NTP servers. That's the job of the ntp daemon. We could perhaps add this configuration to Ansible.

**Comment by antonmolodykh** [comment]
@vpetersson Ok, I will try

**Comment by antonmolodykh** [comment]
This PR is not actual

**Pull request closed without merging**